test/lib/ansible_test/_data/pytest.ini

```diff
@@ -1,3 +1,7 @@
 [pytest]
 xfail_strict = true
 mock_use_standalone_module = true
+# Currently used pytest versions all support xunit2 format too:
```

Comment:
This isn't accurate. This setting has no effect on Python 2.6 (pytest 3.2.5).

Reply:
Right, do you have any suggestions on how to rephrase this?